src/components/Hero/HubHero/index.tsx

```diff
@@ -70,12 +70,16 @@ const HubHero = ({
           </Heading>
           <Text size="lg">{description}</Text>
         </Stack>
-        <HStack justify={{ md: "center", xl: "start" }} spacing="4">
+        <Stack
+          direction={{ base: "column", md: "row" }}
```

Comment:
I don't see the figma file stacking the buttons on smaller res, is it correct? or the figma file should be updated?
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/094b1904-a7ba-4ee7-8865-a4cf290f1e12)

Reply:
@nloureiro shows it off to the side in that file.

![Screenshot_20240531-134242.png](https://github.com/ethereum/ethereum-org-website/assets/65234762/3956b048-9ceb-4273-9fe6-2b3090bfb36b)

Though this file is starting to get a little out of date.

Nuno, I can update the file if you'd like!

Reply:
Ah got it! I think the stacked version is better to handle long i18n words.